unit-tests/src/test/scala/RootPackageTest.scala

```diff
@@ -0,0 +1,3 @@
+object RootPackageTest extends tests.Suite {
```

Comment:
```suggestion
object EmptyPackageTest extends tests.Suite {
```
This whole issue is actually cause by the JVM *empty* package. It's not the root package. If it were the root package, the previous implementation would be correct. Let's use the correct terminology.

Reply:
Done ✅